.github/workflows/release.yml

```
@@ -394,10 +394,218 @@ jobs:
+    runs-on: ubuntu-latest
+    needs: [build-release, provenance]
+    if: always() && needs.build-release.result == 'success'
+
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@08c6903cd8c0fde910a37f88322edcfb5dd907a8 # v5.0.0
+
+      - name: Extract version from tag
+        id: version
+        shell: bash
+        run: |
```

actionlint (1.7.7)

shellcheck reported issue in this script: SC2086:info:8:34: Double quote to prevent globbing and word splitting

(shellcheck)